web-services-app/src/test/java/edu/unc/lib/boxc/web/services/rest/modify/ThumbnailIT.java

```diff
@@ -240,13 +243,33 @@ public void deleteThumbnailNoAccess() throws Exception {
     }
 
     @Test
-    public void deleteThumbnailPidIsNotAFile() throws Exception {
+    public void deleteThumbnailPidIsAWork() throws Exception {
```

Comment:
Related to one of my earlier comments, it could be good to add a test for specifying a work that doesn't have a thumbnail assigned.